autogen

```diff
@@ -161,6 +162,13 @@ Options:
                            add a top-level documentation comment after the
                            license header (default: enabled)
 
+  --no-runline, --no-rnl
+                           Skip adding runline code
+                           to the source files (default: false)
+  --no-code, --no-cod
```

Comment:
No need to abbreviate "code" as "cod" :-)

Not every flag needs a short-form if it's already short.